media/configuration-settings/main.js

```diff
@@ -255,17 +244,56 @@ const buildOptionDom = function(target) {
             valueLabelTag.appendChild(spanTag)
             valueLiTag.appendChild(valueLabelTag)
             nameLiTag.innerText = target.options[i].optionName
-        } else {
+        } else if (typeof target.options[i].optionValue === 'string') {
             // 들어오는 값이 string 값일 경우
             nameLiTag.innerText = target.options[i].optionName
-            const inputTag = document.createElement('input')
-            if (target.options[i].optionValue.trim() !== '') {
-                inputTag.value = target.options[i].optionValue
+            if (target.options[i].optionName === 'input_path') {
+                const inputTag = document.createElement('input')
+                inputTag.disabled = true
+                inputTag.id = target.options[i].optionName
+                if (target.options[i].optionValue.trim() !== '') {
+                    inputTag.value = target.options[i].optionValue
+                }
+                const Btn = document.createElement('button')
```

Comment:
I think it would look better with camelcase

Reply:
i did it in new commit